Userland/Libraries/LibJS/Runtime/TypedArray.h

```diff
@@ -35,6 +35,8 @@ class TypedArrayBase : public Object {
 
     virtual size_t element_size() const = 0;
     virtual String element_name() const = 0;
+    virtual bool put_by_index(u32 property_index, Value value) override = 0;
+    virtual Value get_by_index(u32 property_index, AllowSideEffects = AllowSideEffects::Yes) const override = 0;
```

Comment:
*laughs in object rewrite* (I should finish that)

Reply:
Sorry, I'm making more work for you ^^'

Reply:
s'all good!